LICENSE

```diff
@@ -1,11 +1,25 @@
-Copyright (c) 2020 IETF Trust and the persons identified as the
-document authors.  All rights reserved.
-
-This document is subject to BCP 78 and the IETF Trust's Legal
-Provisions Relating to IETF Documents (https://trustee.ietf.org/
-license-info) in effect on the date of publication of this document.
-Please review these documents carefully, as they describe your rights
-and restrictions with respect to this document.  Code Components
-extracted from this document must include Simplified BSD License text
-as described in Section 4.e of the Trust Legal Provisions and are
-provided without warranty as described in the Simplified BSD License.
+BSD 2-Clause License
```

Comment:
The BSD license is only applicable to code components and not to specs. See https://trustee.ietf.org/
license-info.

Reply:
I was hoping someone notice this. Does it become too complicated if we restructure the code components to their own sub-directory and specs to their own so that we could get two LICENSE files pointing which materials have their corresponding licenses: https://softwareengineering.stackexchange.com/questions/304874/declaring-multiple-licences-in-a-github-project#answer-304895 ?

Reply:
> Does it become too complicated

I would probably answer this affirmatively.

Right now, we don't identify anything in the draft explicitly as a "code component".

The directory `scripts` could of course have its own license file -- none of this goes into the document, so it is useful to be explicit about the copyright (ignoring whether the "threshold of originality" is even reached).

The other potential code component is the ABNF, for which the scripts directory provides ways to extract it from the document, but the repo doesn't actually store the (computed) result.  So there is no directory a license could be attached to.

Reply:
Good reasoning. However isn't the license currently too vague and interpretable?